en/deploy/README.md

```diff
@@ -130,159 +130,106 @@ Your code is now on GitHub. Go and check it out!  You'll find it's in fine compa
 
 # Setting up our blog on PythonAnywhere
 
+## Sign up for a PythonAnywhere account
+
 > **Note** You might have already created a PythonAnywhere account earlier during the install steps – if so, no need to do it again.
 
 {% include "/deploy/signup_pythonanywhere.md" %}
 
 
-## Pulling our code down on PythonAnywhere
+## Creating an API token
 
-When you've signed up for PythonAnywhere, you'll be taken to your dashboard or "Consoles" page. Choose the option to start a "Bash" console – that's the PythonAnywhere version of a console, just like the one on your computer.
+This is something you only need to do once.  When you've signed up for PythonAnywhere, you'll be taken to your dashboard.  Find the link near the top right to your "Accounts" page, then select the tab named "API token", and hit the button that says "Create new API token".
 
-<img src="images/pythonanywhere_bash_console.png" alt="pointing at Other: Bash in Start a new Console" />
+<img src="images/pythonanywhere_create_api_token.png" alt="The API token tab on the Accounts page" />
 
-> **Note** PythonAnywhere is based on Linux, so if you're on Windows, the console will look a little different from the one on your computer.
 
-Let's pull down our code from GitHub and onto PythonAnywhere by creating a "clone" of our repo. Type the following into the console on PythonAnywhere (don't forget to use your GitHub username in place of `<your-github-username>`):
+## Configuring our site on PythonAnywhere
 
-{% filename %}PythonAnywhere command-line{% endfilename %}
-```
-$ git clone https://github.com/<your-github-username>/my-first-blog.git
-```
+Go back to the main PythonAnywhere Dashboard by clicking on the logo, and choose the option to start a "Bash" console – that's the PythonAnywhere version of a command line, just like the one on your computer. 
 
-This will pull down a copy of your code onto PythonAnywhere. Check it out by typing `tree my-first-blog`:
+<img src="images/pythonanywhere_bash_console.png" alt="Pointing at Bash in the New Console section" />
+
+> **Note** PythonAnywhere is based on Linux, so if you're on Windows, the console will look a little different from the one on your computer.
+
+Deploying a web app on PythonAnywhere involves pulling down your code from GitHub, and then configuring PythonAnywhere to recognise it and start serving it as a web application.  There are manual ways of doing it, but PythonAnywhere provide a helper tool that will do it all for you. Let's install it first:
```

Comment:
`s/provide/provides/`, as "PythonAnywhere" seems to be singular, not plural (as seen by the "is" in the note above).

Reply:
yep!